core/templates/components/button-directives/create-activity-button.component.spec.ts

```diff
@@ -165,8 +169,9 @@ describe('CreateActivityButtonComponent', () => {
     expect(component.initCreationProcess).toHaveBeenCalled();
   }));
 
-  it('should create a new exploration automatically if the user' +
-    ' cannot create collections', fakeAsync(() => {
+  it('should create a new exploration automatically if the user cannot create' +
```

Comment:
This one, too -- I don't see anything in the test itself about redirection, either asserting it or setting it up etc., so it's hard for me to understand how the body of the test actually tests the thing in the description (which is the main thing the reviewer is looking for).

Reply:
Made changes